app/components/noteworthy-and-popular-project/template.hbs

```diff
@@ -0,0 +1,7 @@
+<a href="/{{project.id}}">
+    <div class="public-projects-item">
+        <h5>{{project.title}}</h5>
+        <span class="prevent-overflow"><i>{{t 'dashboard.noteworthy.by'}}{{contributors}}</i></span>
+        <p data-title="{{compactDescription}}" data-location="top" class="prevent-overflow" data-original-title="" title="">{{project.description}}</p>
```

Comment:
I assume these `data-` attributes are for this guy:
<img width="225" alt="screen shot 2018-01-22 at 5 01 46 pm" src="https://user-images.githubusercontent.com/6776190/35246854-3041f690-ff96-11e7-9810-2253009bb13f.png">
but the popup is missing locally